apiserver/facades/client/action/run.go

```diff
@@ -6,17 +6,15 @@ package action
 import (
 	"context"
 	"fmt"
-	"strings"
 	"time"
 
-	"github.com/juju/collections/set"
-	"github.com/juju/errors"
-	"github.com/juju/names/v6"
+	jujuerrors "github.com/juju/errors"
```

Comment:
todo: use `juju/internal/errors`.

Reply:
True, but we can use internal/errors for everything else. Makes the changes later easier, especially in new code.